packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -124,22 +145,26 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
   }
 
   getMetadata() {
-    return this.#dispatchAction<IMetadata>('getMetadata');
+    return this.dispatcher?.dispatchTransferAction<IMetadata>('getMetadata') ?? null;
```

Comment:
```suggestion
    return this.dispatcher?.dispatchTransferAction<IMetadata>('getMetadata');
```
What's the reason for possibly coalescing `undefined` to `null`? Normally it seems like we should trust the value received, but I could be missing something.

Reply:
`dispatchTransferAction` can return undefined, but `getMetadata` should return only `IMetadata | null`, hence the coalescing op